packages/client/src/services/credit/creditRfqs.ts

```diff
@@ -72,18 +71,24 @@ export const creditRfqsById$ = creditRfqUpdates$.pipe(
     [boolean, Record<number, RfqDetails>]
   >(
     (acc, [update, instruments, dealers]) => {
-      const rec = acc[1]
+      // need to know when we can release updates from this stream
```

Comment:
looks like a lot of changes here, but just a bit of renaming and simplification of each case
(it used to be a bigger change, but then I discovered I could just use the source stream `creditRfqUpdates$` to drive the new ID for a "trade" or accepted quote)